test_conformance/atomics/test_atomics.cpp

```diff
@@ -247,52 +248,47 @@ int test_atomic_function(cl_device_id deviceID, cl_context context,
     log_info("\t(thread count %d, group size %d)\n", (int)threadSize,
              (int)groupSize);
 
-    refValues = (cl_int *)malloc(typeSize * threadSize);
+    refValues.resize(typeSize * threadSize);
```

Comment:
This isn't wrong per se, but it is allocating more memory than it was before, because the previous code was allocating `typeSize * threadSize` bytes, and the new code is allocating `typeSize * threadSize` integers.

Maybe instead of `std::vector<cl_int>`for  `refValues` we should instead have `std::vector<char>`, like we do for `destItems`?  Or, even better, maybe `std::vector<uint8_t>` or similar?

Reply:
@bashbaug yes you are right, my fault. I will correct it.